src/main/java/org/eeichinger/servicevirtualisation/jdbc/MockResultSetHelper.java

```diff
@@ -64,23 +68,23 @@ public static MockResultSet parseResultSetFromSybaseXmlString(String id, String
         return resultSet;
     }
 
-    private static class DatabaseRow {
+    protected class DatabaseRow {
         final List<String> colNames;
-        final Map<String, String> namedValues;
-        final String[] positionalValues;
+        final Map<String, Object> namedValues;
```

Comment:
Needed to migrate to Object type, but this doesn't change the behaviour for Strings